DESCRIPTION

```diff
@@ -17,11 +17,14 @@ License: LGPL (>= 3)
 Depends:
     R (>= 3.0.0)
 Imports:
+    curl,
+    dplyr,
```

Comment:
Adding three new package dependencies for a single function is unnecessary. Please find a way to accomplish the data manipulations using base R methods.